systers_portal/templates/meetup/add_meetup.html

```diff
@@ -0,0 +1,42 @@
+{% extends "meetup/base.html" %}
+
+{% block title %}
+  Add meetup to {{ meetup_location.name }}
+{% endblock %}
+
+{% block head %}
+<link  href="/static/css/jquery-ui.min-1.11.4.css" rel="stylesheet" type="text/css"/>
+  <link  href="/static/css/jquery-ui-timepicker.0.3.3.css" rel="stylesheet" type="text/css"/>
+{% endblock %}
+
+{% load crispy_forms_tags %}
+{% block meetup_location_page_content %}
+
+  <div class="mt20 mb40"></div>
+  <div class="row ml20">
+    <div class="col-md-12">
+      <h1>Add meetup to {{ meetup_location.name }}</h1>
+      <hr/>
+    </div>
+    <div class="col-md-8">
+      <div class="well">
+        {{ form.media }}
+        {% crispy form %}
+      </div>
+    </div>
+  </div>
+{% endblock %}
+
+{% block scripts %}
+<script src="/static/js/libs/jquery-ui.min-1.11.4.js"></script>
+<script src="/static/js/libs/jquery-ui-timepicker-0.3.3.js"></script>
+  <script type="text/javascript">
```

Comment:
I am trying to use above suggested datepicker, but it is giving error - `unorderable types: NoneType() < datetime.date()` for line https://github.com/systers/portal/blob/develop/systers_portal/meetup/models.py#L46

I think the value that it is returning is NoneType().

I experimented by commenting the `clean_field()`method, then I got enter a valid date and time error - https://gyazo.com/9a2be9944a5137c51d531b1810839f32 

I tried to modify the format of date 
'  $('.datepicker').pickadate({
    format: 'yyyy,mm,dd',
    formatSubmit: 'yyyy,mm,dd',
  })'

But nothing changed. Do you have some suggestions?